### ISSUE TITLE
Add an ASCII "projectsTree" version of "projectsGraph"

### ISSUE THREAD

**Issue by dwijnand** [dwijnand/sbt-project-graph#20]

**Comment by benhutchison** [comment]
yes please, any means of immediate ascii output to sbt console would be appreciated

**Comment by giuscri** [comment]
would you consider leveraging an [external tool](https://metacpan.org/release/TELS/Graph-Easy-0.64)?

**Comment by dwijnand** [comment]
Perhaps.  How would one use `graph-easy`?

**Comment by giuscri** [comment]
It's a Perl module exporting a [command-line API](https://stackoverflow.com/a/3391213/2219670). That can be used in a way similar to what's done [with `dot`](https://github.com/dwijnand/sbt-project-graph/blob/62fcdfaf36bb1592ca837d905615bac5b50dda34/src/main/scala/sbtprojectgraph/SbtProjectGraphPlugin.scala#L29).
